nservicebus/upgrades/6to7/moving-msmq-subscriptions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Migrating Msmq subscription messages
+reviewed: 2017-10-5
+component: MsmqTransport
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 6
+ - 7
+---
+
+When using Msmq Subscription storage in NServiceBus Versions 5.x and Versions 6.x, when a subscription queue is not configured, the subscription storage queue automatically defaulted to "NServiceBus.Subscriptions". However, as this queue name was not endpoint specific, endpoints deployed to the same server could potentially share the same subscriptions queue which is undesirable. 
+
+To rectify this, starting from Version 1.0 and above of NServiceBus.Transport.Msmq, if a queue called "NServiceBus.Subscriptions" is detected, an exception will be thrown to prevent potential loss of messages. 
```

Comment:
>to prevent potential loss of messages. 

This has never been proven. It is only been suggested right?

Reply:
Consider this scenario:
Endpoint starts. Creates a new default subscription queue. However, there are subscribers who are stored in the old "NServiceBus.Subscriptions" queue. When an event is published, some of those subscribers may not get the event. IMO, that's potential loss of message.